console/commands/composer.sh

```diff
@@ -58,8 +67,8 @@ then
     fi
 
     mirror_vendor_host_into_container
-    ${COMMANDS_DIR}/exec.sh composer "$@"
+    ${COMMANDS_DIR}/exec.sh composer ${COMPOSER_WORKDIR_PARAM} "$@"
```

Comment:
According to `composer --help` options should be added at the end. So I would add the parameter after `$@`

```
Composer version 1.7.2 2018-08-16 16:57:12

Usage:
  command [options] [arguments]
```

```suggestion
    ${COMMANDS_DIR}/exec.sh composer "$@" ${COMPOSER_WORKDIR_PARAM} 
```

Reply:
Sure, but below we have following list of **options**:
> Options:
      --xml                      To output help as XML
      --format=FORMAT            The output format (txt, xml, json, or md) [default: "txt"]
      --raw                      To output raw command help
  -h, --help                     Display this help message
  -q, --quiet                    Do not output any message
  -V, --version                  Display this application version
      --ansi                     Force ANSI output
      --no-ansi                  Disable ANSI output
  -n, --no-interaction           Do not ask any interactive question
      --profile                  Display timing and memory usage information
      --no-plugins               Whether to disable plugins.
  **-d, --working-dir=WORKING-DIR  If specified, use the given directory as working directory.**
  -v|vv|vvv, --verbose           Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug

So it looks like it should be at first

Reply:
About renaming `$COMPOSER_WORKDIR_PARAM` to `$COMPOSER_DIR_OPTION` - looks good idea. Will rename

Reply:
> Sorry I do not understand. As far as I know `command [options] [arguments]` means: first `composer` then a `command` and then any of the `options` from this list. Am I missing something here?

Ah, that my mistake. I was looking at help command usage. 
So yes, it's a good idea.